packages/neon_framework/lib/src/utils/request_manager.dart

```diff
@@ -80,71 +80,70 @@ class RequestManager {
+    required DynamiteRawResponse<B, H> rawResponse,
+    required UnwrapCallback<T, DynamiteResponse<B, H>> unwrap,
     bool disableTimeout = false,
   }) async =>
       wrap<T, DynamiteRawResponse<B, H>>(
-        account,
-        cacheKey,
-        subject,
-        () async {
+        account: account,
+        cacheKey: cacheKey,
+        subject: subject,
+        callback: () async {
           await rawResponse.future;
 
           return rawResponse;
         },
-        (rawResponse) => unwrap(rawResponse.response),
-        (data) => json.encode(data),
-        (data) => DynamiteRawResponse<B, H>.fromJson(
+        unwrap: (rawResponse) => unwrap(rawResponse.response),
+        serialize: (data) => json.encode(data),
+        deserialize: (data) => DynamiteRawResponse<B, H>.fromJson(
           json.decode(data) as Map<String, Object?>,
           serializers: rawResponse.serializers,
           bodyType: rawResponse.bodyType,
           headersType: rawResponse.headersType,
         ),
-        disableTimeout,
+        disableTimeout: disableTimeout,
       );
 
   /// Executes a WebDAV request.
-  Future<void> wrapWebDav<T>(
-    Account account,
-    String cacheKey,
-    BehaviorSubject<Result<T>> subject,
-    NextcloudApiCallback<WebDavMultistatus> call,
-    UnwrapCallback<T, WebDavMultistatus> unwrap, {
+  Future<void> wrapWebDav<T>({
+    required Account account,
+    required String cacheKey,
+    required BehaviorSubject<Result<T>> subject,
+    required NextcloudApiCallback<WebDavMultistatus> callback,
+    required UnwrapCallback<T, WebDavMultistatus> unwrap,
     bool disableTimeout = false,
   }) async =>
       wrap<T, WebDavMultistatus>(
-        account,
-        cacheKey,
-        subject,
-        call,
-        unwrap,
-        (data) => data.toXmlElement(namespaces: namespaces).toXmlString(),
-        (data) => WebDavMultistatus.fromXmlElement(xml.XmlDocument.parse(data).rootElement),
-        disableTimeout,
+        account: account,
+        cacheKey: cacheKey,
+        subject: subject,
+        callback: callback,
+        unwrap: unwrap,
+        serialize: (data) => data.toXmlElement(namespaces: namespaces).toXmlString(),
+        deserialize: (data) => WebDavMultistatus.fromXmlElement(xml.XmlDocument.parse(data).rootElement),
+        disableTimeout: disableTimeout,
       );
 
   /// Executes a generic request.
   ///
   /// This method is only meant to be used in testing.
   @visibleForTesting
-  Future<void> wrap<T, R>(
-    Account account,
-    String cacheKey,
-    BehaviorSubject<Result<T>> subject,
-    NextcloudApiCallback<R> call,
-    UnwrapCallback<T, R> unwrap,
-    SerializeCallback<R> serialize,
-    DeserializeCallback<R> deserialize, [
-    // ignore: avoid_positional_boolean_parameters
+  Future<void> wrap<T, R>({
+    required Account account,
+    required String cacheKey,
+    required BehaviorSubject<Result<T>> subject,
+    required NextcloudApiCallback<R> callback,
```

Comment:
```suggestion
    required NextcloudApiCallback<R> requestt,
```

Feel free to also change the typedef name (and docs) if want to keep it aligned.

Reply:
I just removed the entire typedef as I don't find it useful. It's just a simple mapping to anther type with the same generics and nothing else. Also the typedef wasn't used outside the file, so it didn't serve any purpose.